prediction_market_agent/agents/autogen_general_agent/deploy.py

```diff
@@ -50,6 +54,9 @@ def post(self, tweet: str | None) -> None:
             logger.info("No tweet was produced. Exiting.")
             return
 
+        if not self.social_media_handlers:
+            self.load()
```

Comment:
`load` is called automatically in `__init__`, no need to do it here again

Reply:
removed